tests/components/ipma/test_ipma.py

```diff
@@ -0,0 +1,80 @@
+"""The tests for the IPMA weather component."""
```

Comment:
This file is named incorrectly? If it's testing weather, name it `ipma/test_weather.py`.

Make sure you have an `__init__.py` file in this directory and include a test for the config flow.

Reply:
👍